tune/tune.c

```diff
@@ -81,9 +81,9 @@ static int exfat_get_volume_label(struct exfat_blk_dev *bd, off_t root_clu_off)
 		return -1;
 	}
 
-	memset(volume_label, 0, 11);
+	memset(volume_label, 0, VOLUME_LABEL_MAX_LEN);
 	if (exfat_utf16_dec(vol_entry->vol_label, vol_entry->vol_char_cnt*2,
-		volume_label, sizeof(volume_label)) < 0) {
+		volume_label, VOLUME_LABEL_MAX_LEN) < 0) {
```

Comment:
Fourth argument for `exfat_utf16_dec` should be size of buffer specified in third argument.